Implementacija/savetovaliste/app/Controllers/Korisnik.php

```diff
@@ -148,4 +148,10 @@ public function postaviPitanje(){
 		// kada korisnik unese pitanje neka predje na pregled pitanja gde moze da nadje svoje tek dodato pitanje
 		return redirect()->to(site_url("$controller/pregled_pitanja"));		
 	}
+
+	// prikaz profila korisnika ciji smo id prosledili 
+	public function profil($userId){
```

Comment:
- Je l kod mene problem ili ovo ispisuje svaki put svoj profil, nezavisno od id-ja?
- Bez id-ja crashuje
- Takodje bi moglo da ide u BaseController, sto ne bi i gosti mogli da vide tudj profil. Mada, ne mora, svejedno, mozemo i namerno to da ogranicimo

Reply:
Sredjeno sve u novom comitu